scripts/sct_image.py

```diff
@@ -272,7 +272,7 @@ def main(args=None):
                 fname_out.append(add_suffix(fname_in[0], '_' + dim_list[i_dim].upper()))
                 im_out[i_dim].setFileName(fname_out[i_dim])
                 im_out[i_dim].save(verbose=verbose)
-            sct.display_viewer_syntax([fname_out])
+            sct.display_viewer_syntax(fname_out)
```

Comment:
good-- that should not break other stuff because it is inside `-mcs` 👍

Reply:
I believe so